.github/workflows/integration.yaml

```diff
@@ -86,9 +86,6 @@ jobs:
         with:
           python-version: '3.9'
       - run: pip install -r ./src/test/python/requirements.txt
```

Comment:
This one is not yet being removed, as it is used by the Django tests. This will be fixed in a following PR.